src/commands/tree_of.rs

```diff
@@ -88,10 +90,24 @@ pub async fn tree_of(matches: &ArgMatches, repo: Repository, config: &Configurat
                             }
                         )
                     },
-                    &|_, node| format!("label = \"{}\" ", node.1.display_name_version()),
+                    &|_, node| format!("label = \"{}\" ", node.1.clone().display_name_version()),
                 );
 
                 println!("{:?}", dot);
+                Ok(())
+            } else if serial_buildorder {
+                let petgraph: DiGraph<Package, DependencyType> = (*dag.dag()).clone().into();
+
+                let topo_sorted = petgraph::algo::toposort(&petgraph, None)
+                    .map_err(|_| Error::msg("Cyclic dependency found!"))?;
```

Comment:
It might make sense to include the returned `Cycle` (if the package names are included in the output) but then again this error should never occur anyway since we build a DAG (so it doesn't really matter).